rest-api/offline/metrics_pipeline.py

```diff
@@ -271,8 +271,9 @@ def map_answers(reader):
                              question_code != RACE_QUESTION_CODE):
       race_codes = [code_dao.get_code(PPI_SYSTEM, value) for value in race_code_values]
       race = get_race(race_codes)
-      yield(last_participant_id, make_tuple(last_start_time,
-                                               make_metric(RACE_METRIC, str(race))))
+      if race == Race.SKIPPED:
+        race = PMI_SKIP_CODE
```

Comment:
I assume Scott and Eric are OK with getting back the PMI skip code for race and the other fields?

Reply:
Yes, that works.  (John and I chatted about this yesterday.)